doc/sphinx/source/data/commondata.rst

```diff
@@ -0,0 +1,355 @@
+.. _commondata:
+
+=======================
+Experimental data files
+=======================
+
+Data made available by experimental collaborations comes in a variety of
+formats. For use in a fitting code, this data must be converted into a common
+format that contains all the required information for use in PDF fitting.
+Existing formats commonly used by the community, such as in `HepData <https://www.hepdata.net/>`_,
+are generally unsuitable. Principally as they often do not fully describe the
+breakdown of systematic uncertainties. Therefore over several years an NNPDF
```

Comment:
We should also mention that one of the main reasons is that they do not have common format. The breakdown of the systematics are always (at least most of the time) included in the raw data.

Reply:
unless someone gets annoyed due to our use of `often` I think it is fine...